HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/JobDetail/JobDetail.cshtml

```diff
@@ -94,6 +94,10 @@
                                 <dd>@volunteerDetails.EmailAddress</dd>
                                 <dt>@(volunteerDetails.PhoneNumbers().Count() > 1 ? "Phone numbers" : "Phone number")</dt>
                                 <dd>@string.Join(" / ", volunteerDetails.PhoneNumbers())</dd>
+                                <dt>Address</dt>
+                                <dd>
+                                    @Html.DisplayFor(x => volunteerDetails.Address, new { MaskAddress = AgeUk || Hlp, Truncate = true })
```

Comment:
MaskAddress needs to be false (or null) here, otherwise Nikki won't be able to see the volunteer addresses.